tests/queries/0_stateless/02861_replacing_merge_tree_with_block_number_column.reference

```diff
@@ -0,0 +1,13 @@
+== Only last version remains after OPTIMIZE W/ CLEANUP ==
```

Comment:
Test file name should be changed. 
maybe `xxx_replacing_merge_tree_with_vertical_merge_and_cleanup` ?

Reply:
Right